rllib/evaluation/rollout_worker.py

```diff
@@ -722,14 +746,26 @@ def wrap(env):
                 )
                 sample_async = True
             elif isinstance(method, type) and issubclass(method, OffPolicyEstimator):
+                gamma = self.io_context.worker.policy_config["gamma"]
+                # Grab a reference to the current model
+                keys = list(self.io_context.worker.policy_map.keys())
+                if len(keys) > 1:
```

Comment:
Nice catch! I guess multi-agent offline RL would be very very tricky :)

Reply:
Yup, this check was already in the OPE class, we can probably allow multi-agent in a future PR.